fustat23_02

fustat23 opened the pull request:
검토 부탁

fustat23 committed:
fustat23_02

randomkj commented:
asdfdsfsdfas

randomkj commented:
asdasdada

randomkj reviewed (commented):
다르게 보인다 review changes 적는 부분임

fustat23 committed:
fustat23_0412.py 0412올립니다.

fustat23_0412.py 0412올립니다.
fustat23_0412.py 0412올립니다.
fustat23_0412.py 0412올립니다.

randomkj reviewed (commented):
oo

randomkj commented on fustat23.py.py:
```diff
@@ -0,0 +1,20 @@
+
+lines = csv.reader(f1)
+wr = csv.writer(f2)
+
```
sssss

randomkj commented on fustat23.py.py:
```diff
@@ -0,0 +1,20 @@
+
+file_name1 = "weather.csv"
+file_name2 = "weather2.csv"
+f1 = open(file_name1, "r", encoding="utf-8")
```
start

randomkj commented on fustat23.py.py:
```diff
@@ -0,0 +1,20 @@
+import csv
+
+file_name1 = "weather.csv"
```
saaaaaa

randomkj replied:
aaaa

randomkj commented on fustat23.py.py:
```diff
@@ -0,0 +1,20 @@
+lines = csv.reader(f1)
+wr = csv.writer(f2)
+
+header = next(lines)
```
aaaa

randomkj commented on fustat23.py.py:
```diff
@@ -0,0 +1,20 @@
+
+header = next(lines)
+
+for i in range(5) :
```
aaaa

randomkj reviewed (changes requested):
aaaaa

randomkj commented on fustat23.py.py:
```diff
@@ -0,0 +1,20 @@
+import csv
+
+file_name1 = "weather.csv"
```
START REVIEW

fustat23 replied:
이게 무슨 말이지?

fustat23 replied:
햐해결완료

randomkj commented on fustat23.py.py:
```diff
@@ -0,0 +1,20 @@
+import csv
+
+file_name1 = "weather.csv"
+file_name2 = "weather2.csv"
```
REAL START EWVIEW

randomkj commented on fustat23.py.py:
```diff
@@ -0,0 +1,20 @@
+f1 = open(file_name1, "r", encoding="utf-8")
+f2 = open(file_name2, "w", encoding="utf-8", newline="")
+
+lines = csv.reader(f1)
```
REAL START EWVIEW 2

randomkj replied:
FSDFSDFSFSFS

randomkj reviewed (commented):
zhapsxm

randomkj reviewed (changes requested):
request changes

randomkj commented on fustat23_0412.py.py:
```diff
@@ -0,0 +1,20 @@
+import csv
+
+file_name1 = "weather.csv"
+file_name2 = "weather2.csv"
```
fghfhfdhfd

randomkj reviewed (commented):
지금 현재 시각 0308 분 리뷰체인지 코멘트 달면 어디로 들어가는지 확인 해 봅시다

randomkj commented on fustat23.py.py:
```diff
@@ -0,0 +1,20 @@
+import csv
```
지금 현재 시각 0330 분 리뷰체인지 코멘트 달면 어디로 들어가는지 확인 해 봅시다

randomkj commented on fustat23.py.py:
```diff
@@ -0,0 +1,20 @@
+import csv
+
```
지금 현재 시각 0330 분 리뷰체인지 코멘트 달면 어디로 들어가는지 확인 해 봅시다

randomkj commented on fustat23.py.py:
```diff
@@ -0,0 +1,20 @@
+file_name1 = "weather.csv"
+file_name2 = "weather2.csv"
+f1 = open(file_name1, "r", encoding="utf-8")
+f2 = open(file_name2, "w", encoding="utf-8", newline="")
```
지금 현재 시각 0330 분 리뷰체인지 코멘트 달면 어디로 들어가는지 확인 해 봅시다

randomkj reviewed (commented):
지금 현재 시각 0330 분 리뷰체인지 코멘트 달면 어디로 들어가는지 확인 해 봅시다

randomkj commented on fustat23.py.py:
```diff
@@ -0,0 +1,20 @@
+print("파일쓰기 종료!")
+    
+f1.close()
+f2.close()
```
asdasdada

randomkj replied:
asdasda
